src/NodaTime.Test/Calendars/RegularWeekYearRuleTest.cs

```diff
@@ -18,7 +18,7 @@ public void RoundtripFirstDay_Iso7()
             // In the Gregorian calendar with a minimum of 7 days in the first
             // week, Tuesday January 1st -9998 is in week year -9999. We should be able to
             // round-trip.
-            var rule = WeekYearRule.ForMinDaysInFirstWeek(7);
+            var rule = new RegularWeekYearRule(7);
```

Comment:
Done. Made it return RegularWeekYearRule instead of just IWeekYearRule as we may want to expose MinDaysInFirstWeek as a property, for example.